### PR TITLE
added initial screen with tabs for electron app

### PR DESCRIPTION
**Pull request by Rahul139-stack**
**Sample Screen**:
image
image
image

### CODE REVIEW

File: src/views/InitialScreen/index.vue [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        isActive: false,
+      }
+    },
+    methods: {
+      openTab(evt, tabName) {
+        this.activeTab = true
+        const tabs = ["welcome", "intro", "services", "docs"]
+        if (tabs.includes(tabName)) {
+          this.type = tabName
+          console.log("tab", tabName)
```
Comment (Harikarthyk) [inline src/views/InitialScreen/index.vue:69]: remove console log added for testing purposes.

File: src/views/InitialScreen/index.vue [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    <div class="tabcontent">
+      <welcome v-if="type === 'welcome'" />
+      <introduction v-if="type === 'intro'" />
+      <docs v-if="type === 'docs'"></docs>
+      <services v-if="type === 'services'"></services>
+    </div>
+  </div>
+</template>
+<script>
+  import welcome from "./welcome.vue"
```
Comment (Harikarthyk) [inline src/views/InitialScreen/index.vue:42]: Please put the components in component directory

File: src/views/InitialScreen/index.vue [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+  import introduction from "./intro.vue"
+  import docs from "./docs.vue"
+  import services from "./services.vue"
+  export default {
+    name: "InitialScreen",
+    components: {
+      welcome,
```
Comment (Harikarthyk) [inline src/views/InitialScreen/index.vue:49]: All the components must start in capital letter, please follow the standards followed already in the code base,

File: src/views/InitialScreen/services.vue [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+      <h1>Services</h1>
+      <div class="data-view-wrapper">
+        <DataTable
+          :value="data"
+          striped-rows
+          responsive-layout="scroll"
+          :scrollable="true"
+          scroll-height="415px"
+        >
+          <Column field="year" header="Year"></Column>
```
Comment (Harikarthyk) [inline src/views/InitialScreen/services.vue:13]: why is this static ?

File: src/views/InitialScreen/welcome.vue [hunk 5]
```diff
@@ -0,0 +1,24 @@
+<template>
+  <div class="main">
+    <div class="welcome">
+      <img
+        src="/assets/logo.png"
+        alt="Girl in a jacket"
```
Comment (Harikarthyk) [inline src/views/InitialScreen/welcome.vue:6]: What kind of alt message is this.

Please follow some standards.

**Review by Harikarthyk — CHANGES_REQUESTED** [review]
Please follow the previously followed standards.

All the components must be in the components directory, and it should be in capital letters.

And if you take a reference from any other code, please take some time to change it according to our project.

File: src/views/InitialScreen/welcome.vue (hunk 5 above)
Comment (Rahul139-stack) [reply src/views/InitialScreen/welcome.vue:6]: Done

File: src/views/InitialScreen/index.vue (hunk 1 above)
Comment (Rahul139-stack) [reply src/views/InitialScreen/index.vue:69]: Done

File: src/views/InitialScreen/index.vue (hunk 2 above)
Comment (Rahul139-stack) [reply src/views/InitialScreen/index.vue:42]: Done

File: src/views/InitialScreen/index.vue (hunk 3 above)
Comment (Rahul139-stack) [reply src/views/InitialScreen/index.vue:49]: Done

File: src/views/InitialScreen/services.vue (hunk 4 above)
Comment (Rahul139-stack) [reply src/views/InitialScreen/services.vue:13]: Its for demo purpose, services part need to be integrate

File: src/views/InitialScreen/index.vue [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+        tablinks: "",
+        activeTab: false,
+        type: "welcome",
+        isActive: false,
+      }
+    },
+    methods: {
+      openTab(evt, tabName) {
+        this.activeTab = true
+        const tabs = ["welcome", "intro", "services", "docs"]
```
Comment (Harikarthyk) [inline src/views/InitialScreen/index.vue:75]: const can be in caps.

File: src/router/index.ts [hunk 7]
```diff
@@ -18,6 +18,11 @@ const router = createRouter({
         // dialog: Maps,
       },
     },
+    {
```
Comment (Harikarthyk) [inline src/router/index.ts:21]: I think you have used this in place of loader they what is the purpose of adding them as a separate routes.

**Review by Harikarthyk — CHANGES_REQUESTED** [review]
I can see a button hanging on the bottom right on this screen.

image

I could not find any code progress loader for the services.

**Pull request closed without merging**